doc/catalog-format-spec.rst

```diff
@@ -358,9 +358,9 @@ following entries:
    [origin.'case(distribution)']
    'debian|ubuntu' = "native:make"
 
-* ``archive-hash``: mandatory string for source archives. A "kind:digest" field
-  that specifies a hash kind and its value. The only accepted kind is SHA512 at
-  this time.
+* ``origin-hashes``: mandatory string for source archives, optional for git
+  origins. An array of "kind:digest" fields that specify a hash kind and its
+  value. Kinds accepted are sha256, sha384, sha512.
```

Comment:
Why accept multiple kinds? Let's take the most secure.

Reply:
So only sha512 it is.

Reply:
Fixed in latest push